Code/skyrim_ui/src/app/components/server-list/server-list.component.ts

```diff
@@ -48,14 +50,15 @@ export class ServerListComponent {
   clientVersion$: Observable<string>;
 
   formSearch = new FormControl<string>('');
-  rowHeight = 16 * document.documentElement.clientWidth / 1920 * 2;
+  rowHeight = new BehaviorSubject(16)
```

Comment:
replace `16` with a dynamic value or replace the `BehaviorSubject` with a `ReplaySubject`